python/cuml/test/test_naive_bayes.py

```diff
@@ -320,6 +322,86 @@ def test_bernoulli_partial_fit(x_dtype, y_dtype, nlp_20news):
     assert_allclose(y_hat, y_sk)
 
 
+@pytest.mark.parametrize("x_dtype", [cp.float32, cp.float64])
+@pytest.mark.parametrize("y_dtype", [cp.int32, cp.int64])
+@pytest.mark.parametrize("is_sparse", [True, False])
+@pytest.mark.parametrize("norm", [True, False])
+def test_complement(x_dtype, y_dtype, is_sparse, norm, nlp_20news):
+    X, y = nlp_20news
+    n_rows = 500
+
+    X = sparse_scipy_to_cp(X, x_dtype).astype(x_dtype)
+    y = y.astype(y_dtype)
+
+    X = X.tocsr()[:n_rows]
+    y = y[:n_rows]
+    if not is_sparse:
+        X = X.todense()
+
+    sk_model = skComplementNB(norm=norm)
+    cuml_model = ComplementNB(norm=norm)
+
+    sk_model.fit(X.get(), y.get())
+    cuml_model.fit(X, y)
+
+    sk_score = sk_model.score(X.get(), y.get())
+    cuml_score = cuml_model.score(X, y)
+    cuml_proba = cuml_model.predict_log_proba(X).get()
+    sk_proba = sk_model.predict_log_proba(X.get())
+
+    THRES = 1e-3
+
+    assert_array_equal(sk_model.class_count_, cuml_model.class_count_.get())
+    assert_allclose(sk_model.class_log_prior_,
+                    cuml_model.class_log_prior_.get(), 1e-6)
+    assert_allclose(cuml_proba, sk_proba, atol=1e-2, rtol=1e-2)
+    assert sk_score - THRES <= cuml_score <= sk_score + THRES
+
+
+@pytest.mark.parametrize("x_dtype", [cp.float32, cp.float64])
+@pytest.mark.parametrize("y_dtype", [cp.int32,
+                                     cp.float32, cp.float64])
+@pytest.mark.parametrize("norm", [True, False])
+def test_complement_partial_fit(x_dtype, y_dtype, norm, nlp_20news):
+    chunk_size = 500
+    n_rows = 1500
+
+    X, y = nlp_20news
+
+    X = sparse_scipy_to_cp(X, x_dtype).astype(x_dtype)
+    y = y.astype(y_dtype)[:n_rows]
```

Comment:
It might be helpful to set a skewed class distribution and maybe filter out for the first `n` or so occurrences of those classes to test the robustness to class imbalance. Maybe create a dict w/ the final distribution: `{ 'class1': 50, 'class2': 10, 'class3': 8 }`